pkg/util/namespace/namespace.go

```diff
@@ -23,3 +23,70 @@ func IsOpenShiftSystemNamespace(ns string) bool {
 		strings.HasPrefix(ns, "kube-") ||
 		strings.HasPrefix(ns, "openshift-")
 }
+
+// FilteredOpenShiftNamespace returns true if ns is a namespace in the defined hardcoded map
+func FilteredOpenShiftNamespace(ns string) bool {
+	filteredNamespaces := map[string]struct{}{
```

Comment:
can you put these in alphabetical order as well?

Reply:
Hi Ben, thanks for your review feedback, as I mentioned in another reply, we probably need to get some input from @m1kola to clarify the requirement of the work item.

Reply:
It's not the first time I'm seeing this pattern in the repo, so there's probably a reason behind it , but why not use a ```map[string]bool``` instead of ```map[string]struct{}``` ? It seems like the former is more readable and avoids a 
```
_, ok := filteredNamespaces[ns]
return ok

Reply:
The boolean value in the map doesn't provide any additional value to determining if a key exists. Since Golang doesn't have `contains` on an array or slice the use of an empty struct takes up less memory. 

There is this discussion about it which goes deeper: https://gist.github.com/davecheney/3be245c92b61e5045f75 and if you want to read some funny comments on how go doesn't have "array lookups" -> https://stackoverflow.com/questions/10485743/contains-method-for-a-slice

TL;DR, it doesn't really make a huge difference. If we decide `map[string]bool` is easier to read we should have a refactor story to remain consistent in this codebase.

Reply:
But you could still do ```isfiltered, ok := filteredNamespaces[ns]``` even with booleans if you really want to see if a value has been set. Here we are effectively using this as a boolean map, because the only set values are the excluded namespaces.  

Also if we ever want to exclude a namespace from filtering for some reason, we could explicitly set it to false instead of just not putting it in the list.

Reply:
@facchettos - Thanks for your time to review. Yeah, I used the `map[string]bool` in my first commit but got review feedback suggested that I should follow the previous pattern of the codebase to keep it consistent. Since it doesn't make big difference so I agreed to use `map[string]struct{}` for now.

Reply:
@facchettos I agree maybe we should create a work item to convert everything.  For now I think it's fine to leave the changes as @xiaoyu74 has them, but we can convert all `map[string]struct{}` to `map[string]bool` in a follow-up item.